auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -25,6 +25,125 @@
         </div>
       </div>
 
+      <template>
```

Comment:
This might be worth it to make this into a separate component what do the reviewers think?

Reply:
Yes please.

Reply:
Done.